### PR TITLE
 README update

### PR DESCRIPTION
**Pull request by DRA-chaos**
Rectification of a few grammatical errors in the README

### CODE REVIEW

**Comment by adekusar-drl** [comment]
@woodsp-ibm I'd leave these changes to you, if you don't mind.

**Review by woodsp-ibm — COMMENTED** [review]
I will note that the wording of 'you' instead of 'one' and other text is consistent with how these are written throughout Qiskit so I don't think we want to change those. Also other texts are consistent across the repos in how for instance the tutorials are referred to in further examples. There are no grammatical issues I think just proposed changes that go against consistency and the fact that, for instance we refer to the tutorials as a learning path since that is what they were intended as.

File: README.md [hunk 1]
```diff
@@ -2,10 +2,7 @@
...
 
-The Machine Learning package simply contains sample datasets at present. It has some
-classification algorithms such as QSVM and VQC (Variational Quantum Classifier), where this data
-can be used for experiments, and there is also QGAN (Quantum Generative Adversarial Network)
-algorithm.
+The Machine Learning package contains sample datasets, classification algorithms such as QSVMs (Quantum State Vector Machines), VQCs (Variational Quantum CLassifiers) and QGANs (Quantum Generative Adversarial Networks).
```
Comment (woodsp-ibm) [inline README.md:5]: Probably this intro text ought to be brought more up todate, though I can see this reads a little easier than the old text.

File: README.md [hunk 2]
```diff
@@ -15,11 +12,11 @@ We encourage installing Qiskit Machine Learning via the pip tool (a python packa
 pip install qiskit-machine-learning
 ```
 
-**pip** will handle all dependencies automatically and you will always install the latest
+**pip** will handle all dependencies automatically and will always install the latest
```
Comment (woodsp-ibm) [inline README.md:15]: Lets stay with the old text - this is how the other repos, eg Terra talk about this

File: README.md [hunk 3]
```diff
@@ -15,11 +12,11 @@ We encourage installing Qiskit Machine Learning via the pip tool (a python packa
...
 (and well-tested) version.
 
-If you want to work on the very latest work-in-progress versions, either to try features ahead of
-their official release or if you want to contribute to Machine Learning, then you can install from source.
+If one wants to work on the latest work-in-progress versions, either to try features ahead of
```
Comment (woodsp-ibm) [inline README.md:18]: Same here, lets not switch from 'you' to 'one'

File: README.md [hunk 4]
```diff
@@ -28,9 +25,9 @@ To do this follow the instructions in the
 
 ### Optional Installs
 
-* **PyTorch**, may be installed either using command `pip install 'qiskit-machine-learning[torch]'` to install the
-  package or refer to PyTorch [getting started](https://pytorch.org/get-started/locally/). When PyTorch
-  is installed, the `TorchConnector` facilitates its use of quantum computed networks.
+* **PyTorch**, may be installed either using the command `pip install 'qiskit-machine-learning[torch]'` to install the
```
Comment (woodsp-ibm) [inline README.md:28]: The addition of 'the' is ok, seems like it was missed out. Thanks for spotting that. Lets not add text referring to someone as 'one'. Also I think 'its' in the last part is more correct the TorchConnector facilitates the use of quantum networks within/by Torch, the subject of this paragraph, and to which 'it' is referencing

File: README.md [hunk 5]
```diff
@@ -39,7 +36,7 @@ To do this follow the instructions in the
 
 Now that Qiskit Machine Learning is installed, it's time to begin working with the Machine Learning module.
 Let's try an experiment using VQC (Variational Quantum Classifier) algorithm to
-train and test samples from a data set to see how accurately the test set can
+train and test samples from a dataset to see how accurately the test set can
```
Comment (woodsp-ibm) [inline README.md:39]: I think either 'data set' or 'dataset' is fine. However since we use dataset earlier for consistency I guess this works better.

File: README.md [hunk 6]
```diff
@@ -82,7 +79,7 @@ be classified.
 
 ### Further examples
 
-Learning path notebooks may be found in the
+The notebooks can be found in the
```
Comment (woodsp-ibm) [inline README.md:82]: I think I commented in the overall note of the intent/meaning here etc.

**Comment by woodsp-ibm** [comment]
@DRA-chaos did you want to continue with this at all? I think there are a couple of minor grammatical errors that would be left/fixed following my comments. If not then please just close this; otherwise I will close this if I see no further activity in the near future.
